src/main/java/uk/gov/hmcts/reform/cwrdapi/util/RequestUtils.java

```diff
@@ -30,4 +41,35 @@ public static void trimIdamRoles(List<CaseWorkersProfileCreationRequest> request
             }
         });
     }
+
+    public static PageRequest validateAndBuildPaginationObject(Integer pageNumber,
+                                                               Integer pageSize,
+                                                               String sortColumn,
+                                                               String sortDirection,
+                                                               String loggingComponentName,
+                                                               int configPageSize,
+                                                               String configSortColumn) {
+
+        if (pageNumber != null && pageNumber < 0) {
+            log.info("{}:: Invalid Page Number", loggingComponentName);
+            throw new InvalidRequestException(String.format(INVALID_FIELD, PAGE_NUMBER));
+        }
+        if (pageSize != null && pageSize <= 0) {
+            log.info("{}:: Invalid Page Size", loggingComponentName);
+            throw new InvalidRequestException(String.format(INVALID_FIELD, PAGE_SIZE));
+        }
+        if (!StringUtils.isEmpty(sortDirection)) {
+            try {
+                Sort.Direction.fromString(sortDirection);
+            } catch (IllegalArgumentException illegalArgumentException) {
+                log.info("{}:: Invalid Sort Direction", loggingComponentName);
+                throw new InvalidRequestException(String.format(INVALID_FIELD, SORT_DIRECTION));
+            }
+        }
+        return PageRequest.of(pageNumber == null ? 0 : pageNumber,
+                pageSize == null ? configPageSize : pageSize,
```

Comment:
page number and side already check for zero right ?

Reply:
yes, this is Null check on Integer (if request param not sent) to set it to default config value when null